src/utils.ts

```diff
@@ -79,6 +80,30 @@ export async function onDebugStartDisplay(viewsToCheck: string[]) {
   })
 }
 
+function checkInfosetFileExtension(
+  infosetOutput: InfosetOutput,
+  infosetFormat: string
+) {
+  // If the infoset file path doesn't end with the infoset output format, update to end with infoset output format.
+  if (!infosetOutput.path.endsWith(`.${infosetFormat}`)) {
```

Comment:
It looks like there's a couple of other instances where `RegExp` objects are being tested / exec'ed. Exporting this to a function might be helpful for readability and future us.

```ts
let fileExtensionSearchResult = regexTestAndFilter( . . . )
```

Reply:
So this is the only place I believe is filtering off a regex. Other places are using `.exec` and some use `.test`. These perform different as `exec` give you a result and `test` just tells you true or false. I mean we could make wrappers around these but not sure how useful that is as you would just be calling `WrapperFunction` each time instead of `exec` or `test`. We could also make a call back function but that would also kind of just achieve the same thing.

Reply:
@stricklandrbls I can't really add this change as what I am doing with this regexp is very different from others amd I think adding a function like this would basically involve passing the RegExp and then running the `.test` or `.exec` function which I don't think is completely necessary.